.github/workflows/automated-zip-to-pr.yml

```
@@ -0,0 +1,124 @@
+          echo "THEME_NAME=${{ steps.match-theme-name.outputs.group1 }}" >> $GITHUB_ENV
+          echo "BRANCH_NAME=auto/${{ steps.match-theme-name.outputs.group1 }}" >> $GITHUB_ENV
+
+      # Only do these if the target is a COMMENT on the issue
+      - name: Extract Zip URL (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-url-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\((.*?\.zip)\)'
+      
+      - name: Extract Zip Filename (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-filename-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\(.*\/(.*?\.zip)\)'
+
+      - name: Extract Theme Name (comment)
+        uses: actions-ecosystem/action-regex-match@v2
+        if: ${{ github.event.comment.body }}
+        id: match-theme-name-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '\(.*\/(.*?)\.zip\)'
+
+      # We only do this one if the BODY name matches the COMMENT name
+      - name: Define Variables (comment)
+        if: ${{ github.event.comment.body && steps.match-theme-name.outputs.group1 == steps.match-theme-name-comment.outputs.group1 }}
+        run: |
+          echo "ZIP_URL=${{ steps.match-url-comment.outputs.group1 }}" >> $GITHUB_ENV
+          echo "ZIP_FILENAME=${{ steps.match-filename-comment.outputs.group1 }}" >> $GITHUB_ENV
+          echo "THEME_NAME=${{ steps.match-theme-name-comment.outputs.group1 }}" >> $GITHUB_ENV
+          echo "BRANCH_NAME=auto/${{ steps.match-theme-name-comment.outputs.group1 }}" >> $GITHUB_ENV
+
+      # All the rest of these actions will only run if there's a $ZIP_URL defined (otherwise the content doesn't fit our criteria)
+
+      - uses: actions/checkout@v3
+        if: ${{ env.ZIP_URL }}
+        with:
+          fetch-depth: 0
+
+      - name: Download Zip & Push to Branch
+        if: ${{ env.ZIP_URL }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git checkout $BRANCH_NAME || git checkout -b $BRANCH_NAME
+          wget $ZIP_URL
+          unzip -o -qq ./$ZIP_FILENAME -d ./$THEME_NAME
+          sudo chown -R $USER:$USER .
+          sudo rm ./$ZIP_FILENAME
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git config --global --add --bool push.autoSetupRemote true
+          git add --all
+          git commit -m "Automated Theme Change: [$THEME_NAME]"
+          git push origin $BRANCH_NAME
+
+      - name: Create Pull Request
+        if: ${{ env.ZIP_URL }}
+        uses: repo-sync/pull-request@v2
+        with:
+          destination_branch: "trunk"
+          source_branch: "${{ env.BRANCH_NAME }}" 
+          pr_title: "Automated Theme Change: [${{ env.THEME_NAME }}]" 
```

For title, I think its good avoid automation keyword. I suggest something like `Add new theme: THEME_NAME`

```suggestion
          pr_title: "Add new theme: ${{ env.THEME_NAME }}" 
```